### PR TITLE
[TASK] Clean up class imports

### PR DESCRIPTION
**Pull request by NamelessCoder**
Also fixes a single missing doc-comment. Zero active code touched.

### CODE REVIEW

**Comment by bmack** [comment]
Apart from that - looks good.

**Comment by NamelessCoder** [comment]
I used PHPStorm's "clean up imports" feature - there's almost surely a static analysis tool we could use for the same checking in Travis (phpcs probably can do it). I'll look around for a good candidate.

File: src/Core/Parser/BooleanParser.php [hunk 1]
```diff
@@ -6,8 +6,6 @@
  * See LICENSE.txt that was shipped with this package.
  */
 
-use TYPO3Fluid\Fluid\Core\Parser\Exception as ParserException;
```
Comment (lolli42) [inline src/Core/Parser/BooleanParser.php]: this is still used.

Comment (ohader) [inline src/Core/Parser/BooleanParser.php]: This one still is in use in `parseTermToken()`

File: src/Core/Parser/SyntaxTree/Expression/CastingExpressionNode.php [hunk 2]
```diff
@@ -6,9 +6,7 @@
  * See LICENSE.txt that was shipped with this package.
  */
 
-use TYPO3Fluid\Fluid\Core\Parser;
 use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
-use TYPO3Fluid\Fluid\Core\Parser\SyntaxTree\Expression\ExpressionException;
```
Comment (lolli42) [inline src/Core/Parser/SyntaxTree/Expression/CastingExpressionNode.php]: still used.

**Comment by NamelessCoder** [comment]
I'll go over this one again - the reason for the import problems there is I split the changes into two PRs, https://github.com/TYPO3/Fluid/pull/291 and this one, and the removals ended up in the wrong one. CastingExpressionNode removal looks like a similar problem, will fix that too of course.

File: src/Core/Parser/SyntaxTree/Expression/CastingExpressionNode.php (hunk 2 above)
Comment (NamelessCoder) [inline src/Core/Parser/SyntaxTree/Expression/CastingExpressionNode.php]: is in the same scope so the import was redundant

File: src/Core/Parser/BooleanParser.php (hunk 1 above)
Comment (NamelessCoder) [inline src/Core/Parser/BooleanParser.php]: No longer used after https://github.com/TYPO3/Fluid/pull/291

**Comment by NamelessCoder** [comment]
Issues have been addressed, apologies for the confusion!

File: src/Core/Parser/BooleanParser.php (hunk 1 above)
Comment (bmack) [reply src/Core/Parser/BooleanParser.php]: Agreed, now solved

File: src/Core/Parser/SyntaxTree/Expression/CastingExpressionNode.php (hunk 2 above)
Comment (bmack) [reply src/Core/Parser/SyntaxTree/Expression/CastingExpressionNode.php]: yes. reviewed, is fine.

File: src/ViewHelpers/CountViewHelper.php [hunk 3]
```diff
@@ -7,7 +7,6 @@
  */
 
 use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
-use TYPO3Fluid\Fluid\Core\ViewHelper;
```
Comment (bmack) [inline src/ViewHelpers/CountViewHelper.php]: That one is still in use, however!

**Pull request merged**